### DISCUSSION TITLE
[Release] New `@celo/contracts` library to conveniently import (core) contracts using Solidity

### DISCUSSION THREAD

**Discussion by arthurgousset** [celo-org/celo-monorepo#10311] [General]
Hi all 👋

## TLDR

We just released a new **[@celo/contracts](https://www.npmjs.com/package/@celo/contracts)** library. 

We hope this helps you import (core) contracts in Solidity more conveniently and helps you leverage Celo's core contracts more easily. 

This was requested here among other places:
- https://github.com/celo-org/celo-monorepo/issues/8206

## Usage

Here is an example of how you can import core contracts into your solidity contract using `@celo/contracts`:

```java
pragma solidity ^0.5.13;

import '@celo/contracts/common/UsingRegistryV2.sol';

contract Example is UsingRegistryV2 {
  constructor() public {
    require(getAccounts().createAccount());
  }
}
```

## Next steps

Please let us know if you run into any issues or have any feature requests.
All comments and suggestions welcome! 🙏
